infra/aws/terraform/prow-build-cluster/karpenter.tf

```diff
@@ -18,11 +18,15 @@ module "karpenter" {
   source  = "terraform-aws-modules/eks/aws//modules/karpenter"
   version = "~> 20.20"
 
-  cluster_name                    = module.eks.cluster_name
+  cluster_name = module.eks.cluster_name
+
   create_access_entry             = true
   create_pod_identity_association = true
 
-  namespace = "karpenter"
+  node_iam_role_use_name_prefix = false
+  node_iam_role_name            = "karpenter-nodes"
+
+  queue_name = "karpenter-queue"
```

Comment:
will there be more than one karpenter controller per account, per region (thinking about potential naming conflicts). Same for the role name. Injection the cluster name usually helps resolve this

Reply:
No, I don't think so. There are two different accounts and clusters, and I wanted to standardize the naming. These two clusters have different names, so that's why I wanted to avoid injecting the cluster name.